operations/power_management/Shut_Down_and_Power_Off_the_Management_Kubernetes_Cluster.md

```diff
@@ -202,6 +264,33 @@ documentation (`S-8031`) for instructions on how to acquire a SAT authentication
    If the process continues to report errors due to `Failed to stop containers`, then iterate on the above step. Each iteration should reduce the number of containers running. If necessary,
    containers can be manually stopped using `crictl stop CONTAINER`. If containers are stopped manually, then re-run the above procedure to complete any final steps in the process.
 
+1. (`ncn-m001#`) Unload DVS and `Lnet` kernel modules from worker nodes.
+
+   > This step helps to avoid error messages in the console log while Linux is shutting down similar to "DVS: task XXX exiting on a signal"
+
+   ```bash
+   pdsh -w $WORKERS 'lsmod | egrep "^dvs\s+"; rm -rf /run/dvs; \
+      echo quiesce / > /sys/fs/dvs/quiesce; modprobe -r dvs; sleep 5; \
+      modprobe -r dvsipc dvsipc_lnet dvsproc; lsmod | egrep "^lnet\s"; \
+      lsmod | egrep "^lustre\s"; systemctl stop lnet; lsmod | egrep "^lnet\s"'
+   ```
+
+1. (`ncn-m001#`) Adjust boot order for management NCNs so the next boot will use disk.
+   This ensures that when the node is powered up again it will boot from disk rather than attempting
+   to PXEboot before the services to support that are available.
```

Comment:
I would probably just put a space here and then you can probably avoid adding the spelling exception.

```suggestion
   to PXE boot before the services to support that are available.
```

Reply:
fixed